### PR TITLE
Add Sentry On-Premise

### PR DESCRIPTION
**Pull request by kaveet**
Adding Sentry was proposed a while back (see #616), but was rejected due to duplication in `awesome-sysadmin`. It has since been removed from that repository (see https://github.com/n1trux/awesome-sysadmin/pull/199) due to license incompatibility.

Sentry now uses BSD-3-Clause (https://github.com/getsentry/sentry/blob/master/LICENSE) so I felt that it would be appropriate here.

Happy to move it to another section or close this PR if it doesn't fit in here.

---

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [X] Your additions are ordered alphabetically.
- [X] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [X] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [X] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [X] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [X] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), incluing closed ones.
- [X] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [X] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1153,7 +1153,9 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 
 ### Bug Trackers
 
-See **[Ticketing](#ticketing)**
+See also **[Ticketing](#ticketing)**
+
+- [Sentry On-Premise](https://github.com/getsentry/onpremise) - A powerful error tracking platform with wide language support and a robust API. ([Source Code](https://github.com/getsentry/sentry)) `BSD-3-Clause` `Python/Django`
```
Comment (nodiscc) [inline README.md]: Please move it to https://github.com/Kickball/awesome-selfhosted#ticketing

**Comment by kaveet** [comment]
@nodiscc Moved to ticketing as requested.

Thanks!

**Comment by Kickball** [comment]
I don't think I properly understand the licensing/copyright situation that lead to it being removed from [n1trux/awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin/). Has the licensing situation since changed or are only parts of the Sentry repository opensource and non-copyrighted?

**Comment by kaveet** [comment]
The PR I linked from awesome-sysadmin (https://github.com/n1trux/awesome-sysadmin/pull/199) appeared to remove Sentry on the premise that its license did not permit redistribution. As far as I can tell from the license, that's not the case: https://github.com/getsentry/sentry/blob/master/LICENSE -- redistribution is (and has been) permitted as long as their disclaimer is included.

I don't entirely understand why it was removed either.

**Pull request merged**